distribution/docs/pom.xml

```diff
@@ -45,6 +45,10 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+        <properties>
```

Comment:
Looks like there's extra spacing here... I _think_ this will fail the format check.

Reply:
fixed